tailord/src/fancontrol/runtime.rs

```diff
@@ -36,7 +36,9 @@ impl FanRuntimeData {
 
             tokio::select! {
                 _ = tokio::time::sleep(delay) => {},
-                _ = process_suspend(&mut self.suspend_receiver) => {}
+                _ = process_suspend(&mut self.suspend_receiver) => {
+                    self.fan_speed = self.io.get_fan_speed_percent(0).unwrap();
```

Comment:
is there a better way to handle the error case here than unwrap?

Reply:
We could handle this case and maybe we should, but an error would indicate a problem with the driver ioctl interface, which would mean that something is going really really bad.